scripts/modules/b2b-account/quotes.js

```diff
@@ -75,16 +80,79 @@ define([
                     });
                 }
             }
+            if (!self.model.get("b2bAccounts")) {
+                var b2bAccount = new B2BAccountModels.b2bAccounts({ pageSize: 200 });
+                b2bAccount.apiGet().then(function (accounts) {
+                    self.model.set("b2bAccounts", accounts);
+                    self.render();
+                });
+            }
+            
+            $('[data-mz-action="applyfilter"]').on('keyup input', function(e) {
+                e.preventDefault();
+                clearTimeout(timeout);
+                FILTERSTRING = "";
```

Comment:
You don't need to set filterString to empty string here. It's always cleared in `filterGrid`.

Reply:
Done

Reply:
done